ranger/ranger.go

```diff
@@ -289,3 +314,19 @@ func (m Metadata) templateFunc() (string, func(key string) string) {
 }
 
 type ShutdownFn func(context.Context) error
+
+func maintModeHandler(p template.Parser, l logger.Logger, contact string) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		w.Header().Add("Retry-After", "300")
+
+		tmpl, err := p.Parse("tmpl/maintenance.tmpl")
```

Comment:
Skipped the responder as `resp.Html` is too heavy with it checking flashes and other aspects of an "app" response.